docs/scos/dev/front-end-development/migration-guide-upgrade-to-webpack-v5.md

```diff
@@ -10,258 +11,256 @@ This document provides instructions for upgrading Webpack to version 5 in your S
-       "terser-webpack-plugin": "~5.3.6",
-       "webpack": "~5.74.0",
-       "webpack-merge": "~5.8.0"
-   }
-   ```
-
-   - If Merchant Portal is used:
-
-     ```json
-     {
-         "@types/webpack": "~5.28.0"
-     }
-     ```
-
-2. To update and install package dependencies, run:
-
-   ```bash
-   rm -rf node_modules
-   npm install
-   ```
+1. In `package.json`, update the following dependencies to the new version:
+
+```json
+{
+    "clean-webpack-plugin": "~4.0.0",
+    "compression-webpack-plugin": "~10.0.0",
+    "copy-webpack-plugin": "~11.0.0",
+    "mini-css-extract-plugin": "~2.7.2",
+    "optimize-css-assets-webpack-plugin": "~6.0.1",
+    "postcss": "~8.4.20",
+    "terser-webpack-plugin": "~5.3.6",
+    "webpack": "~5.74.0",
+    "webpack-merge": "~5.8.0"
+}
+```
+
+- If Merchant Portal is used:
+
+```json
+{
+    "@types/webpack": "~5.28.0"
+}
+```
+
+2. Update and install package dependencies:
+
+```bash
+rm -rf node_modules
+npm install
+```
    
-   Ensure that the `package-lock.json` file and the `node_modules` folder have been updated.
+Ensure that the `package-lock.json` file and the `node_modules` folder have been updated.
 
 ## 2) Update configuration files
 
-**Do the following changes to update configuration files:**
+To update configuration files, make the following changes:
 
 1. `frontend/configs/development-watch.js`:
 
-   ```js
-   const merge = require('webpack-merge');
-   // must be
-   const { merge } = require('webpack-merge');
-   ```
+```js
+const merge = require('webpack-merge');
+// must be
+const { merge } = require('webpack-merge');
+```
 
 2. `frontend/configs/development.js`:
-
    - Rename the `jsonpFunction` property to the `chunkLoadingGlobal` of the `output` object.
    - Rename the `vendors` property to the `defaultVendors` of the `cacheGroups` object.
 
 3. `frontend/configs/production.js`:
-
-   ```js
-   const merge = require('webpack-merge');
-   // must be
-   const { mergeWithCustomize, customizeObject } = require('webpack-merge');
-   ```
-
-   ```js
-   const mergeWithStrategy = merge.smartStrategy({
-       plugins: 'prepend',
-   });
-   // must be
-   const mergeWithStrategy = mergeWithCustomize({
-       customizeObject: customizeObject({
-           plugins: 'prepend',
-       })
-   });
-   ```
-
-   ```js
-   plugins: [
-       new CompressionPlugin({
-           filename: '[path].gz[query]',
-       }),
-   
-       new BrotliPlugin({
-           asset: '[path].br[query]',
-           test: /\.js$|\.css$|\.svg$|\.html$/,
-           threshold: 10240,
-           minRatio: 0.8
-       })
-   ]
-   // must be
-   plugins: [
-       new CompressionPlugin({
-           filename: '[path][base].br[query]',
-           algorithm: 'brotliCompress',
-           test: /\.(js|css|html|svg)$/,
-           threshold: 10240,
-           minRatio: 0.8,
-       }),
-   ]
-   ```
-
-   Remove the `cache` property in the `new TerserPlugin({ ... })` minimizer plugin.
-
```

Comment:
@VadymSachenko Removing indents impairs readability.

Reply:
@FRITS-Kh from the perspective of UI, it does not have any impact. The content is displayed identically (with and without indents)

Reply:
We work in the `Markdown` format and that makes sense for the format.
You can check it for example in GitHub or IDE preview.
Maybe the rule should be adjusted?

Reply:
@FRITS-Kh, we must be consistent. Unfortunately, we can't do this. If we adjust this one, we must redo all the rest (we talk about hundreds of documents) but the reason for doing so must be stronger than just making the documents a bit prettier on GitHub. Also, idents might cause issues in Algolia, that's another reason we avoid them.

Reply:
@FRITS-Kh Though we can apply indents for sub-steps because, in this case, they are displayed differently (right under the sub-step).